packages/react/src/tag-group/tag-group.tsx

```diff
@@ -0,0 +1,143 @@
+import { Close } from '@obosbbl/grunnmuren-icons-react';
+import { cva, cx } from 'cva';
+import type { RefAttributes } from 'react';
+import {
+  Button,
+  Tag as RACTag,
+  TagGroup as RACTagGroup,
+  type TagGroupProps as RACTagGroupProps,
+  TagList as RACTagList,
+  type TagListProps as RACTagListProps,
+  type TagProps as RACTagProps,
+} from 'react-aria-components';
+
+const tagVariants = cva({
+  base: [
+    'inline-flex cursor-pointer items-center gap-2 rounded-lg font-medium text-sm transition-colors duration-200',
+    //Focus
+    'focus-visible:outline-focus-offset [&:not([data-focus-visible])]:outline-none',
+    //Border
+    'shadow-[inset_0_0_0_2px_#002169]',
+    //Backgrounds
+    'data-[hovered]:!bg-sky bg-white text-black aria-selected:bg-sky-light data-[allows-removing]:bg-sky-light',
+    //Icons
+    '[&_svg]:h-4 [&_svg]:w-4',
+  ],
+});
+
+export type TagGroupProps = Omit<RACTagGroupProps, 'className'> &
+  RefAttributes<HTMLDivElement> & {
+    /**
+     * CSS classes to apply to the tag group
+     */
+    className?: string;
+    /**
+     * The function to call when the tag is removed
+     */
+    onRemove?: (key: React.Key) => void;
+
+    /**
+     * The selection mode for the tag group
+     * Override this so that "none" isn't an option
```

Comment:
Kan vi slette "Override this so....". Det lekker ut til konsumenten som en del av dokumentasjonen for prop. Føles internt ut.